microcosm_elasticsearch/registry.py

```diff
@@ -67,13 +67,8 @@ def createall(self, force=False, only=(), skip=()):
             if force and index.exists():
                 index.delete()
                 index.flush(ignore_unavailable=True)
-            try:
-                index.create()
-                index.refresh()
-            except RequestError as error:
```

Comment:
The expectation is that by using `index.save` we won't get an "index already exists" exception, making this block unnecessary.

Reply:
I was under the impression that updating mappings in place doesn't work terribly well and can result in some odd behaviors (such as multi-typed mappings).

Reply:
Interesting, were you able to test locally to try to add/remove fields from models? Curious to see how the mappings update.

Reply:
@saivickna I tested for my latest changes on frigg (https://github.com/globality-corp/frigg/pull/38) and it worked fine.
@jessemyers interesting, I'll try gathering more info on that issue; it would be much better not having to reindex the data every time we add a mapping field.